src/validations/editorValidation.ts

```diff
@@ -29,9 +29,18 @@ export const useGetAllErrors = async (
     const isLoggedInUMLS = await Promise.resolve(CheckLogin());
     const [validatedCodes, translationResults, valuesetsErrors] =
       await Promise.all([
-        ValidateCustomCqlCodes(customCqlCodes, isLoggedInUMLS.valueOf()),
+        ValidateCustomCqlCodes(
+          customCqlCodes,
+          isLoggedInUMLS.valueOf(),
+          cqlResult?.using?.name
+        ),
+
         TranslateCql(cql),
-        GetValueSetErrors(cqlResult.valueSets, isLoggedInUMLS.valueOf()),
+        GetValueSetErrors(
+          cqlResult.valueSets,
+          isLoggedInUMLS.valueOf(),
+          cqlResult?.using?.name
```

Comment:
I see this value is what's getting passed down for `model`, which is then compared to "QDM". 
So this value would come out to be either `QDM`, `FHIR` or `QI-Core`? It would not include version numbers?
Also, I think Joe and Greg were hoping to avoid logic like "if QDM" and "if FHIR" in a bunch of places. I wonder if there's any way to accomplish this based on the other strategy being discussed?

Reply:
Yes, `cqlResult?.using?.name` would be the model that is being used in cql(e.g. `using QDM version '5.6'`), it would not include version. As far as I know, I do not think value set validation would be based on the model & version. it would be only on the model. Except for getting the oids for value sets and code systems, I think our editor is pretty much model agnostic. So, not really sure if we should invest in separating the model-specific functionality for the editor. But I can update if we want to follow a different approach.

Reply:
okay, moved model specific code to madie-utility(will be used in editor and patient spa)